test/decoders/test_video_decoder_ops.py

```diff
@@ -39,6 +39,16 @@
 torch._dynamo.config.capture_dynamic_output_shape_ops = True
 
 
+def needs_cuda(test_item):
```

Comment:
I suspect we'll need it in other places, so move it to https://github.com/NicolasHug/torchcodec/blob/main/test/utils.py ?

Also please add small description comment e.g.

```suggestion
def needs_cuda(test_item):
    # Decorator for skipping CUDA tests when CUDA isn't available
```